src/components/BulkUploadListItem.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { CheckCircleIcon, XCircleIcon } from '@heroicons/react/24/outline';
+import { BulkUpload } from '../interfaces/BulkUpload';
+import './BulkUploadListItem.css';
+import { Spinner } from './Spinner';
+
+interface BulkUploadListItemProps {
+  upload: BulkUpload;
+}
+
+export const BulkUploadListItem = ({ upload }: BulkUploadListItemProps) => {
+  const {
+    fileName, fileSize, status, createdAt,
+  } = upload;
+
+  // TODO: Normalize these against the statuses returned by the API.
+  const isProcessing = status === 'processing';
+  const isComplete = status === 'complete';
+  const hasError = status === 'error';
+
+  const makeClassNames = () => {
+    const classNames = ['bulk-upload-list-item'];
+
+    classNames.push(status);
+
+    return classNames.join(' ');
+  };
```

Comment:
I'm probably going to inline this to `['bulk-upload-list-item', status].join(' ')` or so.

Reply:
Yeah, I was considering doing the same, but wasn't sure how the status normalizing would work out / if we'd need to do more `switch`ing here. Feel free to do that.